admin_manual/installation/source_installation.rst

```diff
@@ -270,7 +270,7 @@ Additional Apache Configurations
 Pretty URLs
 -----------
 
-Pretty URLs are created automatically when ``.htaccess`` is writable by the 
+Pretty URLs are created automatically when ``.htaccess`` is readable by the 
```

Comment:
That is incorrect. The HTTP user needs to write the htaccess because we will add some more details there.